dapr/clients/grpc/_request.py

```diff
@@ -64,3 +65,49 @@ def data(self) -> GrpcAny:
     @property
     def content_type(self) -> Optional[str]:
         return self._content_type
+
+
+class InvokeBindingRequestData:
+    """A request data representation for invoke_binding API.
+
+    This stores the request data and metadata with the proper serialization.
+    This seralizes data to bytes and metadata to a dictionary of key value pairs.
+
+    Attributes:
+        data (bytes): the serialized data for invoke_binding request.
```

Comment:
Just want to understand why we don't mention that it takes string as well?

Reply:
good catch